BikeData/src/main/scala/by/itechart/internship/dao/DatabaseReader.scala

```diff
@@ -0,0 +1,15 @@
+package by.itechart.internship.dao
+
+import by.itechart.internship.entities.{TripInfo, TripInfoView}
+import by.itechart.internship.logic.MyPostgresDriver.api._
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent._
+
+object DatabaseReader {
```

Comment:
There's a commonly used pattern called Data Access Object (DAO) :
https://en.wikipedia.org/wiki/Data_access_object
It's common in Java/Scala projects

What you have here is DAO, so I'd rename DatabaseReader  into TripInfoDao

Another approach is ActiveRecord (more common in Ruby community):
https://en.wikipedia.org/wiki/Active_record_pattern